Lib/test/test_typing.py

```diff
@@ -4554,6 +4554,15 @@ class Commentable(Protocol):
         )
         self.assertIs(type(exc.__cause__), CustomError)
 
+    def test_deferred_evaluation_of_annotations(self):
+        class DeferredProto(Protocol):
+            x: DoesNotExist
+        self.assertEqual(get_protocol_members(DeferredProto), {"x"})
+        self.assertEqual(
```

Comment:
Question: when should we test all formats and when just one is enough?

Reply:
I guess in this case Protocol is not doing anything very sophisticated with its annotations, so we can just run a basic test to make sure it works.